test/adding_tests.md

```diff
@@ -237,11 +237,14 @@ For example to create a `Configuration` object that uses a certain docker image
 with a randomized name:
 
 ```go
-var names test.ResourceNames
-names.Config := test.AppendRandomString('hotdog', logger)
-_, err := clients.ServingClient.Create(test.Configuration(namespaceName, names, imagePath))
-if err != nil {
-    return err
+func TestSomeAwesomeFeature(t *testing.T) {
```

Comment:
Another think we can do is add `crd_test.go` with contents along the lines of: https://github.com/knative/pkg/blob/master/test/helpers/data_test.go. This will generate an "Example" section of the godoc (presuming anyone looks at out godocs: https://godoc.org/github.com/knative/pkg/test/helpers).

Reply:
I'll leave it here, I think it may be more visible than the godoc since people are reading code in this directory first before writing tests (hopefully :D)